server/src/metadata.rs

```diff
@@ -155,6 +180,24 @@ pub mod entities_metadata_query {
                     .and_then(|title| title.as_str())
                     .map(|title| title.to_string());
                 let id = get_iri(result.id as i32);
+
+
+                let authors_map: HashMap<String, String> = serde_json::from_value(result.authors.unwrap_or_default())
+                        .unwrap_or_default();
+
+                let creators: Vec<Creator> = authors_map.iter()
+                    .map(|(id, username)| Creator {
+                        creator_type: CreatorType::Person,
+                        // Id is a url that links to our authors. It looks like
+                        // the following
+                        // https://serlo.org/user/:userId/:username e.g
+                        // https://serlo.org/user/240298/felix_eccardt
+                        id: Some(format!("https://serlo.org/user/{}/{}", id, username)),
```

Comment:
```suggestion
                        id: get_iri(id),
```

Actually `https:://serlo.org/<user-id>` works -> Having the username in the ID is also not a good idea since it might change (we sometimes rename users).